packages/markdown/index.js

```diff
@@ -202,12 +202,12 @@ export function react(text, opts = {}, components = {}) {
         'rdme-pin': PinWrap,
         table: Table(sanitize),
         a: Anchor(sanitize),
-        h1: Heading(1, count),
-        h2: Heading(2, count),
-        h3: Heading(3, count),
-        h4: Heading(4, count),
-        h5: Heading(5, count),
-        h6: Heading(6, count),
+        h1: Heading(1, count, opts),
+        h2: Heading(2, count, opts),
+        h3: Heading(3, count, opts),
+        h4: Heading(4, count, opts),
+        h5: Heading(5, count, opts),
+        h6: Heading(6, count, opts),
```

Comment:
This passes in the full RDMD options object, yeah?

Reply:
Yeah.